static/app/views/issueDetails/traceTimeline/traceTimeline.spec.tsx

```diff
@@ -181,19 +199,17 @@ describe('TraceTimeline', () => {
     });
     MockApiClient.addMockResponse({
       url: `/organizations/${organization.slug}/events/`,
-      body: emptyBody,
+      body: discoverBody,
       match: [MockApiClient.matchQuery({dataset: 'discover', project: -1})],
     });
-    // I believe the call to projects is to determine what projects a user belongs to
+    // Used to determine the project badge
     MockApiClient.addMockResponse({
       url: `/organizations/${organization.slug}/projects/`,
       body: [],
     });
 
     render(<TraceTimeline event={event} />, {
-      organization: OrganizationFixture({
-        features: ['related-issues-issue-details-page', 'global-views'],
```

Comment:
We don't need `related-issues-issue-details-page` since it will be the new default.

`'global-views'` is already defined in the `organization` mock, thus, I do not need to pass it in here again.